# pick_place.action

# Goal definition
# all the coords wrt to base_footprint (Tiagos's base)
int32 id


---
# Result definition
float64[] objects_coords   #vector of doubles like {x1, y1, z1}
float64[] obstacles_coords   #vector of doubles like {x1, y1, x2, y2, x3, y3, x4, y4}
string status


---
# Feedback definition
string status
